src/api/TKMember.h

```diff
@@ -485,6 +485,16 @@
        onSuccess:(OnSuccessWithToken)onSuccess
          onError:(OnError)onError;
 
+/**
+ * Looks up a existing access token where the calling member is the grantor and given member is
+ * the grantee.
+ *
+ * @param toMemberId beneficiary of the active access token
+ */
+- (void)getActiveAccessToken:(NSString *)toMemberId
```

Comment:
Not sure get active access token is a good naming. For transfer we have 
getTransfer: and getTransfers:
we might want to make them consistent. Could be in another pr.

Maybe getAccessToken: and GetAccessTokensFeed: would be clearer?

Reply:
I'm not sure if an active access token is analogous to a transfer so I don't think consistency with transfer naming is relevant. I'm open to better naming suggestions but I'm not sure getAccessToken or getAccessTokenFeed represents the intent (lookup an existing access token between two users) in a clearer manner.